src/garage/torch/modules/mlp_module.py

```diff
@@ -1,17 +1,17 @@
-"""
-MLP Module.
+"""MLP Module.
 
 A Pytorch module composed only of a multi-layer perceptron (MLP), which maps
 real-valued inputs to real-valued outputs.
 """
 
-from torch import nn as nn
+import copy
+
+from torch import nn
 from torch.nn import functional as F  # NOQA
```

Comment:
you can remove this as soon as #1013 is submitted

Reply:
fixed